Makefile

```diff
@@ -133,7 +133,7 @@ e2e: cleanup-e2e install ## Run e2e test locally. e.g. make e2e test=replica_set
 e2e-gh: ## Trigger a Github Action of the given test
 	scripts/dev/run_e2e_gh.sh $(test)
 
-cleanup-e2e:
+cleanup-e2e: ## Cleans up e2e test env
```

Comment:
Do we need this comment? Does it bring anything?

Reply:
yes!
The make target helper generates a help target comment with it i.e.
```
❯ make help | rg "Cleans up"
  cleanup-e2e                Cleans up e2e test env
```

But it does need an explicit comment for it